fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1370,6 +1382,10 @@ public static boolean isChainedParameter(String name) {
         return name.contains(SearchConstants.CHAINED_PARAMETER_CHARACTER);
     }
 
+    public static boolean isReverseChainedParameter(String name) {
```

Comment:
It's actually a code. 
```suggestion
    public static boolean isReverseChainedParameter(String code) {
```

Reply:
renamed to code